test/svg-test-helper.js

```diff
@@ -110,60 +110,54 @@ export const getPathCommandsFromContainer = (container) => {
   return parseSvgPathCommands(commandStr);
 };
 
-const exhibitsShapeSequence = (commandString, shapeSeqeuence) => {
+export const exhibitsShapeSequence = (commandString, shapeSeqeuence) => {
   const commands = parseSvgPathCommands(commandString);
   return commands.every(
     (command, index) => command.name === shapeSeqeuence[index]
   );
 };
 
-export const svgExpectations = {
-  /**
-   * Asserts that the provided command string produces a rectangular shape.
-   *
-   * @param {string} commandString - the "d" attribute of a `path` element.
-   * @returns {void}
-   */
-  expectIsRectangular(commandString) {
-    const isBar = exhibitsShapeSequence(commandString, RECTANGULAR_SEQUENCE);
-    expect(isBar).toEqual(true);
-  }
-};
+/**
+ * Retrieve the raw svg height of a bar.
+ *
+ * @param {string} commandString - The "d" attribute of a `path` element.
+ * @returns {Number}             - The height of the bar in svg units.
+ */
+export const getBarHeight = (commandString) => {
+  const commands = parseSvgPathCommands(commandString);
 
-export const getShapeUtils = {
```

Comment:
Functions are just being exported from this file directly, no longer grouped under objects.